### PR TITLE
Onseekmediaclocktimer enableseek/setmediaclocktimer enable seek value wrong type

### PR DESCRIPTION
**Pull request by OVikhrov**
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua [hunk 1]
```diff
@@ -0,0 +1,108 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- INVALID_DATA
```
Comment (aarotar) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua]: @OVikhrov, please update Requirement summary and add related to SetMediaClockTimer functional requirement also

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+              endTime =
+              {
+                hours = 0,
+                minutes = 5,
+                seconds = 35
+              },
+              updateMode = "COUNTUP",
+              enableSeek = "string"
+            })
+
```
Comment (aarotar) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua]: @OVikhrov, plase avoid redundant empty lines and use lua beautifier

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua]: Accepted

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Value_Type.lua]: Accepted

**Pull request merged**